content/integrations/github.md

```diff
@@ -14,18 +14,41 @@ Capture GitHub commits in Datadog to:
   * Identify when new code changes lead to system alerts or build failures
   * Discuss code changes with your team in the Datadog Event Stream
 
-
+{{< img src="git_integration_screen.png" alt="Git event" >}}
 
 ## Configuration
 
+1. Add this URL to your Github Webhook, **Settings>Webhooks>Add new Webhook** with this URL:
+```
+https://app.datadoghq.com/intake/webhook/github?api_key=<YOUR_DATADOG_API_KEY>
+```
 
-Select 'Github' [on the account settings screen][1] and link your account. You can then select which repos you would like to integrate, which branches, and if you'd like to receive commits and/or issues.
+2. Select what you want to send to datadog, we support:
+     * Push (new commits), 
+     * Create and delete (for tags), 
+     * Pull requests,
+     * Issues, 
+     * All comments.
 
-## What to Expect
+3. Add the branches you wish to monitor for each repository. Wildcards are also supported to add all repositories for a user or Organization.
```

Comment:
"For each repository, add the branches you wish to monitor. If you want to add all repositories for a user or organization, use wildcards." (what's an example of using wildcards?)